### PR TITLE
[CIGolang]support ssh on winc

### PR DESCRIPTION
**Pull request by kuiwang02**
/cc @Xia-Zhao-rh 
cc @jfrancoa

### CODE REVIEW

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.13-debug-winc-aws-ipi

**Comment by kuiwang02** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-c2s-ipi-disconnected-private-p2-f7

**Comment by kuiwang02** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-byo-iam-role-p2-f14

**Comment by kuiwang02** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.13-debug-winc-aws-ipi

**Comment by kuiwang02** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-byo-iam-role-p2-f14

**Comment by kuiwang02** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-gcp-ipi-disconnected-private-p3-f28

**Comment by kuiwang02** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-sc2s-ipi-disconnected-private-p2-f7

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13.yaml [hunk 1]
```diff
@@ -186,7 +186,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       FORCE_SUCCESS_EXIT: "no"
       TEST_FILTERS: ~ChkUpgrade&;~ConnectedOnly&;Smokerun&
-      TEST_SCENARIOS: Windows_Containers
```
Comment (jfrancoa) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13.yaml:189]: Please, remember to restore back the Windows_Containers scenario ;-)

Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13.yaml:189]: yes. after debugging is done, I will recover it.

**Comment by kuiwang02** [comment]
@jfrancoa could you please add `/lgtm` if you are ok on it? thanks

and then you could try your PR on openshift-tests-private.

**Comment by jfrancoa** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-release-4.13-debug-winc-aws-ipi

**Pull request merged**